README.md

```diff
@@ -1,54 +1,5 @@
+#  Building Clima requires that you have an OpenWeatherMap API key stored within a Secrets.xcconfig file within your project. Make sure you have your API key stored within a Secrets.xcconfig file before running your app.
+Create a file called Secrets.xcconfig in the root directory of your project. Add the following to the file:
+OpenWeatherMapAPIKey = <API key goes here>
 
-![App Brewery Banner](Documentation/AppBreweryBanner.png)
-
-#  Clima
-
-## Our Goal
-
-It’s time to take our app development skills to the next level. We’re going to introduce you to the wonderful world of Application Programming Interfaces (APIs) to grab live data from the internet. If you’re dreaming of making that Twitter-powered stock trading app then you’re about add some serious tools to your toolbelt!
-
-
-## What you will create
-
-By the end of the module, you will have made a beautiful, dark-mode enabled weather app. You'll be able to check the weather for the current location based on the GPS data from the iPhone as well as by searching for a city manually. 
-
-## What you will learn
-
-* How to create a dark-mode enabled app.
-* How to use vector images as image assets.
-* Learn to use the UITextField to get user input. 
-* Learn about the delegate pattern.
-* Swift protocols and extensions. 
-* Swift guard keyword. 
-* Swift computed properties.
-* Swift closures and completion handlers.
-* Learn to use URLSession to network and make HTTP requests.
-* Parse JSON with the native Encodable and Decodable protocols. 
-* Learn to use Grand Central Dispatch to fetch the main thread.
-* Learn to use Core Location to get the current location from the phone GPS. 
-
-### Condition Codes
-```
-switch conditionID {
-        case 200...232:
-            return "cloud.bolt"
-        case 300...321:
-            return "cloud.drizzle"
-        case 500...531:
-            return "cloud.rain"
-        case 600...622:
-            return "cloud.snow"
-        case 701...781:
-            return "cloud.fog"
-        case 800:
-            return "sun.max"
-        case 801...804:
-            return "cloud.bolt"
-        default:
-            return "cloud"
-        }
-```
-
->This is a companion project to The App Brewery's Complete App Development Bootcamp, check out the full course at [www.appbrewery.co](https://www.appbrewery.co/)
-
-![End Banner](Documentation/readme-end-banner.png)
+#  Add Secrets.xcconfig to your repo's .gitignore to prevent it (or other secrets you may have in your app) from being committed to GitHub.
```

Comment:
Other developers don't need to do this; you're correctly committing this change to the `.gitignore` file as part of this PR, so the developer won't need to follow this step. You can delete this line.

Reply:
Just wasn't thinking properly--of course they won't need to do the gitignore part!

Reply:
Certainly! FYI, As of right now, the final line of the Readme, starting with `# Add Secrets.xcconfig . . .` is still in your Readme.md